src/my_project/tagging/image_scoring.py

```
@@ -0,0 +1,149 @@
+            w_hash, w_hue = 0.3, 0.7
+        else:
+            w_hash, w_hue = (
+                ImageScorer.FINAL_WEIGHTS["hash"],
+                ImageScorer.FINAL_WEIGHTS["hue_hist"],
+            )
+
+        final_score = w_hash * hash_score + w_hue * hist_score
+        logging.info(f"""   hash-score = {hash_score}
+                            histogram-score = {hist_score}
+                            total-score = {final_score}
+                            """)
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)